app/controllers/rademade_admin/model_controller.rb

```diff
@@ -19,10 +19,11 @@ class ModelController < RademadeAdmin::AbstractController
     before_action :load_options, :model, :pagination_variants
     before_action :sortable_service, :only => [:index]
 
-    def create
+    def create(&block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `create(*)` if you want the method to accept any arguments but don't care about them.